package.json

```diff
@@ -5,6 +5,27 @@
   "engines": {
     "node": ">= 4"
   },
+  "keywords": [
+    "ui",
+    "uswds",
+    "components",
+    "library",
+    "open source",
+    "united states",
+    "usa",
+    "u.s.",
+    "government",
+    "websites"
+  ],
+  "module": "./dist/js/uswds.min.js",
+  "exports": {
+    "import": "./dist/js/uswds.min.js",
+    "default": "./dist/js/uswds.min.js"
+  },
+  "types": "./index.d.ts",
+  "files": [
+    "dist"
+  ],
```

Comment:
Does this change the guidance in [README](https://github.com/uswds/uswds/blob/develop/README.md#javascript)?

Reply:
Currently, no. When we created a true modularized package, yes. This actually is only to satisfy skypack's criteri. The ESM config and Mappings do nothing for SPA users right now. Since we don't have true Modular endpoints.